### PR TITLE
Eve/suppress complex

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/module/core/regular/converter.hpp [hunk 1]
```diff
@@ -104,7 +104,7 @@ template<scalar_value T> using converter_type = decorated<convert_to_<T>()>;
 //!   namespace eve
 //!   {
 //!      template< eve::value T >
-//!      as_wide_as<T, float> float32(T x) noexcept; //1
+//!      as_wide_as<T, float> float32(T x) noexcept;
```
Comment (jfalcou) [inline include/eve/module/core/regular/converter.hpp:107]: there is still a //2 so wjy removing the // 1 ?

File: include/eve/module/core/regular/is_infinite.hpp [hunk 2]
```diff
@@ -35,15 +35,14 @@ namespace eve
...
+//!     * `x` :  [real argument](@ref eve::value).
 //!
 //!   **Return value**
 //!
 //!     For real entries the call `is_infinite(x)` is semantically  equivalent to
 //!     `eve::is_equal(eve::abs(x), eve::inf(as(x))`,
 //!     in particular this is always false for integral types.
 //!
-//!     For complex entries  `is_infinite(z)` is semantically  equivalent to
 //!     `eve::is_infinite(real(z)) || eve::is_infinite(imag(z))`,
```
Comment (jfalcou) [inline include/eve/module/core/regular/is_infinite.hpp:46]: why not removing the whole sentence ?

**Review by jfalcou — COMMENTED** [review]
Some docs has been modified a bit roughly :p
should we move mandelbrot to kyosu ?

File: include/eve/module/core/regular/converter.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/core/regular/converter.hpp:107]: done

File: include/eve/module/core/regular/is_infinite.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/module/core/regular/is_infinite.hpp:46]: done

**Pull request merged**